publication-commons/src/test/resources/features/NotAllowedAccessToDynamoWithoutUserCredentials.feature

```diff
@@ -0,0 +1,44 @@
+Feature:
+  DynamoDBPublicationService does not allow reading, inserting or modifying an entry without
+  the caller of the service specifying the username who is performing the action.
+
+
+  Background:
+    Given that PublicationService exists
+
+  Scenario: PublicationService requires username for reading a publication
+    Given that PublicationService provides a READ method for reading a single publication
+    And that READ method is being called by a module or an application
+    When the READ call does not include a username
+    Then the module or the application cannot call the READ method
```

Comment:
Or the read only produces results with a match on username?

Reply:
This will be on the features regarding the returning results

Reply:
If the Publication is PUBLISHED, then anyone can read it